src/pred_set/src/pred_setScript.sml

```diff
@@ -4951,6 +4951,14 @@ val SUM_IMAGE_PERMUTES = store_thm(
   `SIGMA (f o g) s = SIGMA f (IMAGE g s)` by SRW_TAC[][SUM_IMAGE_INJ_o] THEN
   SRW_TAC[][]);
 
+Theorem SUM_IMAGE_ADD:
```

Comment:
Seems unused elsewhere so it's not clear that this is the right orientation for the equality.

Reply:
I don't mind either way - `EXTREAL_SUM_IMAGE_ADD` and `REAL_SUM_IMAGE_ADD` have it the other way around so I will change it here to match.